space.go

```diff
@@ -150,3 +181,10 @@ func spaceV1Fields() (schema.Fields, schema.Defaults) {
 	}
 	return fields, defaults
 }
+
+func spaceV2Fields() (schema.Fields, schema.Defaults) {
```

Comment:
So this method is added.
But we still don't use it, eg we still have this code in import v1
```
	fields, defaults := spaceV1Fields()
	fields["id"] = schema.String()
```

Reply:
sure, added V3 and used this func